webrender/src/texture_cache.rs

```diff
@@ -112,6 +112,8 @@ struct CacheEntry {
     eviction_notice: Option<EvictionNotice>,
     // The type of UV rect this entry specifies.
     uv_rect_kind: UvRectKind,
+    // If true the cache entry may be evicted if unused for a number of frames.
```

Comment:
comments needs an update. Also, why is it not a doc comment?

Reply:
It's not a doc comment because the other comments were not doc comments either. Does it make a difference for non-public fields?

Reply:
I secretly hope that we can (one day) generate a private documentation for a module/crate, where those will be useful. In any way, there isn't any downside of using doc comments here, but you aren't required to do the change since you were just following the old code.

Reply:
I don't mind doing the change either in light of your sercet hope. I'll update it.